python/paddle/fluid/initializer.py

```diff
@@ -95,10 +95,15 @@ class ConstantInitializer(Initializer):
     Examples:
         .. code-block:: python
 
-    	    import paddle.fluid as fluid
-            x = fluid.data(name="data", shape=[8, 32, 32], dtype="float32")
-	    fc = fluid.layers.fc(input=x, size=10,
-    		param_attr=fluid.initializer.Constant(value=2.0))
+            import numpy as np
+            import paddle
+            import paddle.nn as nn
+
+            paddle.disable_static()
+            linear = nn.Linear(32, 64, weight_attr=nn.initializer.Constant(value=2.0))
+            data = np.random.uniform( -1, 1, [30, 10, 32] ).astype('float32')
+            data = paddle.to_tensor ( data )
```

Comment:
Please use `yapf` to format these python code

Reply:
The modified python code is already been formatted by yapf.